backend/api.py

```
@@ -1509,22 +1513,31 @@ async def open_file(body: dict, request: Request, _=Depends(verify_local_request
+        logger.info("[DEBUG-OPEN] File not found!")
         raise HTTPException(status_code=404, detail="File not found")
 
+    logger.info("[DEBUG-OPEN] Attempting to launch file...")
     try:
         import subprocess
         import platform
         
+        logger.info(f"[DEBUG-OPEN] Platform: {platform.system()}")
         if platform.system() == 'Windows':
+            logger.info("[DEBUG-OPEN] Calling os.startfile...")
             os.startfile(file_path)
```

Ruff (0.15.14)

[error] Starting a process without a shell

(S606)